tests/acquia_lift.test

```diff
@@ -483,6 +483,7 @@ class AcquiaLiftsWebTest extends DrupalWebTestCase {
 
     $this->drupalGet('node/' . $post_id);
     $this->assertLiftMetaTag('content_type', 'article');
+    $this->assertLiftMetaTag('context_language', 'en');
```

Comment:
I would use `fr` instead of `en` because `en` is Drupal default.

Reply:
`testNonNodePage` needs to assert `context_language` as well.

Reply:
As discussed prior, this is tested functionality in D7, grabbing the current language so only en works as an assertion. Added to test node page